cohere/client.py

```diff
@@ -220,6 +221,10 @@ def detokenize(self, tokens: List[int]) -> Detokenization:
         })
         return Detokenization(_future=self._executor.submit(self.__request, json_body, cohere.DETOKENIZE_URL))
 
+    def whisper(self, file_paths: List[str]) -> Whispers:
+        response = self.__file_request(file_paths, cohere.WHISPER_URL)
```

Comment:
Handle error when file_paths is not a list of strings

Reply:
What is the best way to check typing. You can't check if it's a list of strings with isInstance easily.

Reply:
Yes
```
if isinstance(value, List):
```